### PR TITLE
OSSM-8536: Add procedure section title

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Affects:

[service-mesh-docs-main](https://github.com/openshift/openshift-docs/tree/service-mesh-docs-main)
[service-mesh-docs-3.0.0tp1](https://github.com/openshift/openshift-docs/tree/service-mesh-docs-3.0.0tp1)

This PR is part of the standalone doc set for the Service Mesh 3.0 project. Kathryn is aware that this content applies for a product that is part of a Technology Preview release. The project is seeking feedback from early adopters.

PR must be CP'd back to the tp1 branch.

Version(s): Tech Preview

Issue: https://issues.redhat.com/browse/OSSM-8536

Link to docs preview:
https://85933--ocpdocs-pr.netlify.app/openshift-service-mesh/latest/install/ossm-installing-openshift-service-mesh.html#deploying-book-info_ossm-about-bookinfo-application

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by pbajjuri20 — APPROVED** [review]
LGTM

File: modules/ossm-deploying-bookinfo-application.adoc [hunk 1]
```diff
@@ -82,7 +84,7 @@ reviews-v3-5b9bd44f4-44hr6       2/2     Running   0          65s
 +
 When the `Ready` columns displays `2/2` the proxy sidecar was successfully injected. Confirm that `Running` appears in the `Status` column for each pod.
 
-. Verify that the `bookinfo` application running by sending a request to the `bookinfo` page. Run the following command at the CLI:
+. Verify that the `bookinfo` application running by sending a request to the `bookinfo` page. Run the following command:
```
Comment (lahinson) [inline modules/ossm-deploying-bookinfo-application.adoc:87]:
```suggestion
. Verify that the `bookinfo` application is running by sending a request to the `bookinfo` page. Run the following command:
```

File: modules/ossm-deploying-bookinfo-application.adoc [hunk 2]
```diff
@@ -82,7 +84,7 @@ reviews-v3-5b9bd44f4-44hr6       2/2     Running   0          65s
 +
 When the `Ready` columns displays `2/2` the proxy sidecar was successfully injected. Confirm that `Running` appears in the `Status` column for each pod.
```
Comment (lahinson) [inline modules/ossm-deploying-bookinfo-application.adoc:85]:
```suggestion
When the `Ready` columns displays `2/2`, the proxy sidecar was successfully injected. Confirm that `Running` appears in the `Status` column for each pod.
```

**Review by lahinson — COMMENTED** [review]
Looks good! Just a couple of minor suggestions.

File: modules/ossm-deploying-bookinfo-application.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-deploying-bookinfo-application.adoc:85]: done

**Pull request merged**